src/App.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
```

Comment:
- [ ] Your README is not descriptive and does not define your application, offer instructions on deployment, etc. Kindly use this [template](https://github.com/microverseinc/readme-template) to make your README provide instructions that will assist anyone who clones your repository.